src/styles/components/_card.scss

```diff
@@ -0,0 +1,136 @@
+.icon_height{
```

Comment:
Some of the class names are in snake case, some camel case, some pascal case and others hyphenated. Any reason for that?

Reply:
They have all been corrected to follow bootstrap's convention

Reply:
Great!!